white_paper.Rmd

```diff
@@ -462,7 +460,11 @@ Operating Procedures.
 
 ### Package Remediation / Testing
 
-One way of mitigating risk for medium/high risk packages is to generate
```

Comment:
I'm not sure this is the place to make recommendations about the risk levels.  We can address separately.